lib/tests/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # Setup compiling the test executable. C++ compile flags are inherited from the parent directory.
-include_directories(${LEATHERMAN_CATCH_INCLUDE})
+include_directories(../inc ${LEATHERMAN_CATCH_INCLUDE} ${Boost_INCLUDE_DIRS})
```

Comment:
Oh, the changes here aren't needed, they're inherited from the parent (lib/CMakeLists.txt).